hack/deploy/build-single-manifests.sh

```diff
@@ -8,7 +8,7 @@ REPO_ROOT=$(dirname ${BASH_SOURCE})/../..
 
 cd $REPO_ROOT
 
-kustomize build config/base > deploy/single-v2/all-in-one-dbless.yaml
-kustomize build config/variants/postgres > deploy/single-v2/all-in-one-postgres.yaml
-kustomize build config/variants/enterprise > deploy/single-v2/all-in-one-enterprise-dbless.yaml
-kustomize build config/variants/enterprise-postgres > deploy/single-v2/all-in-one-enterprise-postgres.yaml
+${REPO_ROOT}/bin/kustomize build config/base > deploy/single-v2/all-in-one-dbless.yaml
```

Comment:
drive-by comment: can we also ensure that the `deploy/` be a deep copy of `deploy-v2` upon autogeneration?

I totally understand this is outside of the scope, but maybe worth taking care of right now as it's so cheap to do.

Reply:
I'm sorry I'm not sure I understand this comment, could you help me please with a re-phrase?

Reply:
I mean that `build-single-manifests.sh` only writes to the `single-v2` directory. And after you run the script, the content of `single/` is uncontrolled - it requires a manual copy that can easily be missed by the person doing the change to `single-v2`.

What I'm suggesting is that this script writes not only to `single-v2`, but also to `single`.

Reply:
Ah, I see. I might be confused, but as per previous conversations I understood the situation was that we are _intentionally_. @rainest might have some thoughts on this

Reply:
We're intentionally holding back and keeping `single` static for the moment, as there's old link references to there. I think if we're going to change this, we should do it as part of some separate PR which is geared just towards that. Is that agreeable?